definitions/ext-vpc_network/definition.yml

```diff
@@ -19,6 +19,8 @@ synthesis:
 
 goldenTags:
   - sample_rate
+  - account_id
```

Comment:
is this one different to `accountId`? is this the aws/integration account id?

Reply:
@nr-jbiggley can you verify this?